tailwind/plugins/base/base.ts

```diff
@@ -31,5 +31,8 @@ export const base = ({ addBase, config }: { addBase: Function, config: Function
         display: 'none',
       },
     },
+    'b, strong': {
+      fontWeight: config('theme.fontWeight.bold'),
+    },
```

Comment:
Sometimes the editors add bold style in the WYSIWYG to headings and links which unintentionally made them font weight 900. This makes them just bold (so the unintentional extra bold headings/links looks like other headings/links)